### PR TITLE
OVAL for controller_rotate_kubelet_server_certs

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by shawndwells** [comment]
@openscap-ci retest this please

File: applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml [hunk 1]
```diff
@@ -5,11 +5,12 @@ title: 'Ensure that the RotateKubeletServerCertificate argument is set'
...
-    <tt>/etc/kubernetes/manifests/kube-controller-manager.yaml</tt> on the
-    master node(s) and set the <tt>--feature-gates</tt> parameter to include
+    <tt>/etc/origin/master/master-config.yaml</tt> on the
+    master node(s) and set the <tt>controllerArguments</tt> parameter to include
     <tt>RotateKubeletServerCertificate=true</tt>. For example:
-
-    <pre>--feature-gates=...,RotateKubletServerCertificate=true,...</pre>
+    <pre>kubernetesMasterConfig:
+      controllerArguments:
+        - RotateKubeletServerCertificate=true</pre>
```
Comment (redhatrises) [inline applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml]: This isn't right. This is a `feature-gate`. See https://docs.openshift.com/container-platform/3.10/admin_guide/disabling_features.html

Comment (shawndwells) [reply applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml]: thanks!

**Comment by shawndwells** [comment]
Just realized we need to update these to look for cluster-wide config settings *and* node-based config maps =/

**Comment by shawndwells** [comment]
On 11/1/18 12:37 PM, Jan Černý wrote:
>
> I was wondering why the build fails. It fails because of wrong 
> indentation on this line, there is one space missing.
>
Eagle eyes. Thanks!

fixed & rebased

File: applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml [hunk 2]
```diff
@@ -26,15 +28,14 @@ rationale: |-
...
+    <tt>controllerArguments</tt> options
 
 ocil: |-
     Run the following command on the master node(s):
-    <pre>$ ps -ef | grep kube-controller-manager</pre>
+    <pre>$ grep RotateKubeletServerCertificate</pre>
 
-    Verify that the <tt>RotateKubletServerCertificate</tt> argument exists
-    and is set to <tt>true</tt>.
+   The output should return <tt>true</tt>.
```
[the remark this answers is not in the document]
Comment (shawndwells) [reply applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml]: fixed

**Comment by jan-cerny** [comment]
Thank you!

**Pull request merged**